spec/system/volunteer_edits_case_spec.rb

```diff
@@ -26,5 +26,10 @@
     check "Court report submitted"
     click_on "Update CASA Case"
     has_checked_field? :court_report_submitted
+
+    expect(page).to have_text("Court Date")
+    expect(page).not_to have_text("Day")
+    expect(page).not_to have_text("Month")
+    expect(page).not_to have_text("Year")
```

Comment:
Let's also update the `spec/system/supervisor_edits_case_spec.rb` spec.

Reply:
Added supervisor test... and it turned out the admin test was being skipped... I've updated that so both the admin test cases (edit and back out of edit) now pass :partying_face: